### PR TITLE
set default route for no classless

### PR DESCRIPTION
**Pull request by giggsoff**
In case of switch network instance we may not have staticroutes, but we [should](https://github.com/lf-edge/eden/pull/591#issuecomment-826658857) define default gateway.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/xen-tools/initrd/udhcpc_script.sh [hunk 1]
```diff
@@ -81,8 +81,14 @@ case "$1" in
     # configure interface and routes
     ip addr flush dev $interface
     ip addr add ${ip}/${mask} dev $interface
-    # shellcheck disable=SC2086
-    [ -n "$router" ] && [ -n "$staticroutes" ] && install_classless_routes $staticroutes
+    # shellcheck disable=SC2086,SC2154
```
Comment (eriknordmark) [inline pkg/xen-tools/initrd/udhcpc_script.sh:84]: Isn't that the wrong thing? Set router on line 7 seems better, no?

**Review by eriknordmark — COMMENTED** [review]
Nit on yetus fix.

File: pkg/xen-tools/initrd/udhcpc_script.sh (hunk 1 above)
Comment (giggsoff) [reply pkg/xen-tools/initrd/udhcpc_script.sh]: Sure, thank you

**Review by rvs — APPROVED** [review]
LGTM

**Review by gkodali-zededa — APPROVED** [review]
LGTM

**Comment by zed-sadov** [comment]
LGTM:
https://github.com/lf-edge/eden/pull/591#issuecomment-828267005

**Pull request merged**